internal/tiltfile/config/config_test.go

```diff
@@ -459,6 +459,34 @@ print(config.tilt_subcommand)
 	require.Equal(t, "foo\n", f.PrintOutput())
 }
 
+func TestTiltfilePath(t *testing.T) {
+	f := NewFixture(t, model.UserConfigState{}, "foo")
+	defer f.TearDown()
+
+	f.File("Tiltfile", `
+print(config.tiltfile_path)
+`)
+
+	_, err := f.ExecFile("Tiltfile")
+	require.NoError(t, err)
+
+	require.Equal(t, f.JoinPath("Tiltfile")+"\n", f.PrintOutput())
+}
+
+func TestTiltfileDir(t *testing.T) {
+	f := NewFixture(t, model.UserConfigState{}, "foo")
+	defer f.TearDown()
+
+	f.File("Tiltfile", `
+print(config.tiltfile_dir)
+`)
+
+	_, err := f.ExecFile("Tiltfile")
+	require.NoError(t, err)
+
+	require.Equal(t, f.Path()+"\n", f.PrintOutput())
+}
+
 func NewFixture(tb testing.TB, userConfigState model.UserConfigState, tiltSubcommand model.TiltSubcommand) *starkit.Fixture {
```

Comment:
it'd be good to have tests that check the value of `config.tiltfile_path`/dir from inside a `load`ed Tiltfile

Reply:
what about entry_path / entry_dir? or maybe main_path / main_dir ?

was trying to find analogues to this in python but was not coming up with good ones.

Reply:
done